docs/source/basic_tutorials/monitoring.md

```diff
@@ -0,0 +1,75 @@
+# Monitoring TGI server with Prometheus and Grafana dashboard
+
+TGI server deployment can easily be monitored through a Grafana dashboard, consuming a Prometheus data collection. Example of inspectable metrics are statistics on the effective batch sizes used by TGI, prefill/decode latencies, number of generated tokens, etc.
+
+In this tutorial, we look at how to set up a local Grafana dashboard to monitor TGI usage.
+
+![Grafana dashboard for TGI](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/tgi/grafana.png)
```

Comment:
I kind of like this graph too:

<img width="1777" alt="Screenshot 2024-05-16 at 16 36 20" src="https://github.com/huggingface/text-generation-inference/assets/204321/9f1560c0-fb7b-435f-9515-a34b16e26444">

It's not super useful to detect core issues, but it's more stable metrics (and what people tend to boast within benchmarks)